e2e-tests/native/wdio-browserstack.conf.ts

```diff
@@ -7,7 +7,7 @@ const getCapability = () => {
   }
   const capability = capabilities[capabilityName]
 
-  if (!capability) {
+  if (capability === undefined) {
```

Comment:
:upside_down_face: Wouldn't it (although it should never be null) more robust if we stay with checking like before?